### ISSUE TITLE
Updates needed?

### ISSUE THREAD

**Issue by erindorothy** [lihas/NeuroPy#13]
It seems this library is not updated to work with Python3. Could I fork it and make some updates? I am working on a group project that uses MindWave Mobile and I'm interested in contributing to this library as a result.

**Comment by lihas** [comment]
Yes, please use it the way you want. I shared the library in the hope that it will help others. Now it has been many years, and the library is outdated. I am not able to find time to update it.

**Comment by odemiray** [comment]
Sorry for bumping this, but any updates about Python 3 version?

**Comment by erindorothy** [comment]
I took interest in your repo because at the time I was working on a class project with a group with Neurosky headsets. At the time, this repo connected Python to Neurosky the best from anything else we could find. We figured out that because Python3 was not backwards compatible enough with Python2 libraries that supported Bluetooth, we could not connect to the headset for more than a split second. 

I encourage you to try to do this again, and use the work we started on my forked version of your repo. I think recent developments with Python3 would make this project happen. As far as I know, all our work was pushed to my fork. Please let me know if you need any clarification on our work. I can look back and piece things together for you.

**Comment by odemiray** [comment]
Thank you! I will be working on it during next two weeks, since i only have access to device during my internship. Will let you know if i stuck on something. And I'll check your fork for some inspirations. Cheers!

**Comment by lihas** [comment]
Hi all,
Sorry for not updating it.
Since I have been seeing that so many people use it, I really want to update it. 
However I do not even know if my hardware still works.
Have been extremely limited by time.
If you have made any improvements to this library feel free to share on your respective accounts.
People will find it and use yours, as you have used mine.
My code belongs to whoever wants to use it. It does not belong to me. I claim no copyrights.
